src/common/components/sidebar/outline-view.js

```diff
@@ -76,20 +76,20 @@ function OutlineView({ outline, onNavigate, onOpenLink, onUpdate}) {
 		}
 	}
 
-	function handleKeyDown(event) {
-		let { key } = event;
-
-		let list = [];
-		function flatten(items) {
-			for (let item of items) {
-				list.push(item);
-				if (item.items && item.expanded) {
-					flatten(item.items);
-				}
+	function flatten (items, list = []) {
```

Comment:
Hm, thank you, that's a good point. It definitely is enabled, as I can see eslint issues in `ZoteroPane` but none in the reader though... So that's something to figure out! For this and other eslint issues below.

Reply:
Yes, that was exactly it! While trying to figure it out, I temporarily changed the path to `babel.config.js` from relative to absolute (just to see) and everything worked. So the small syntax errors should now be taken care of. I rebased and did a new commit to make it easier to fix my edits.